components/Question/AskQuestionForm.tsx

```diff
@@ -177,18 +232,65 @@ function AskQuestionForm({ post, user, onExit }: AskQuestionFormProps) {
           required
         />
       </div>
+      {!post && (
+        <div className={css(styles.researchcoinContainer)}>
+          <div className={css(styles.researchcoinTitle)}>
+            <div>
+              <div className={css(styles.rscLabel)}>ResearchCoin Bounty</div>
+              <p style={{ fontSize: 16 }}>
+                Incentivize the community to answer your question by adding RSC.
+              </p>
+            </div>
+            {withBounty ? (
+              <div onClick={() => setWithBounty(false)}>
+                <Button size="small" customButtonStyle={styles.removeBountyBtn}>
+                  <FontAwesomeIcon icon={faPlus} style={{ marginRight: 4 }} />
+                  Remove Bounty
+                </Button>
+              </div>
+            ) : (
+              <div onClick={() => setWithBounty(true)}>
+                <Button size="small" customButtonStyle={styles.addBountyBtn}>
+                  <FontAwesomeIcon icon={faPlus} style={{ marginRight: 4 }} />
```

Comment:
nit: `icon` should be `faTimes` if `withBounty === true`

Reply:
Good call